README.md

```diff
@@ -1304,6 +1304,7 @@ Table of Contents
   * [framer.com](https://www.framer.com/) - Framer helps you iterate and animate interface ideas for your next app, website, or product—starting with powerful layouts. For anyone validating Framer as a professional prototyping tool: unlimited viewers, up to 2 editors, up to 3 projects.
   * [freeforcommercialuse.net](https://freeforcommercialuse.net/) — FFCU Worry-free model/property release stock photos
   * [Gradientos](https://www.gradientos.app) - Makes choosing a gradient fast and easy.
+  * [Iconfont](https://www.iconfont.cn/) – Download massive custom icons. Translating icons into Fonts. Project Collaboration Tools. Communication Platform.
```

Comment:
we can only add english sites to this list, is there a english version?

Reply:
English translation is not complete

Reply:
![image](https://user-images.githubusercontent.com/3794090/199140358-472c2bca-38f0-4f55-ad10-17baf628b6c1.png)

Reply:
> 我们只能将英文网站添加到此列表中，有英文版本吗？



> we can only add english sites to this list, is there a english version?

https://www.iconfont.cn/?lang=en-us